scripts/convert_user_roles/convert_user_to_team_member_roles.py

```diff
@@ -0,0 +1,102 @@
+import os
+import requests
+
+OPSLEVEL_API_TOKEN = os.environ["OPSLEVEL_API_TOKEN"]
+OPSLEVEL_ENDPOINT = "https://app.opslevel.com/graphql"
+
+LIST_USERS_QUERY = """
+    query roles($endCursor: String) {
+      account {
+        users(filter: {key: role, arg: "user", type: equals}, after: $endCursor) {
+          nodes {
+            id
+            email
+            name
+            role
+          }
+          pageInfo {
+            hasNextPage
+            endCursor
+          }
+        }
+      }
+    }
+    """
+
+UPDATE_USER_ROLE_MUTATION = """
+    mutation update_user_role($email: String, $role: UserRole) {
+      userUpdate(user: {email: $email}, input: {role: $role}) {
+        user {
+          id
+          email
+          name
+          role
+        }
+        errors {
+          message
+          path
+        }
+      }
+    }
+    """
+
+def opslevel_graphql_query(query, variables=None):
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": f"Bearer {OPSLEVEL_API_TOKEN}",
+    }
+    data = {"query": query, "variables": variables}
+    response = requests.post(OPSLEVEL_ENDPOINT, json=data, headers=headers)
+    if response.status_code != 200:
+        raise Exception(f"OpsLevel request failed: {response.content.decode()}")
+    return response.json()
+
+def fetch_users():
+    """Fetch list of users from OpsLevel."""
+    cursor = None
+    has_next_page = True
+    users = []  # Store fetched users
+    while has_next_page:
+        response = opslevel_graphql_query(
+            LIST_USERS_QUERY, variables={"endCursor": cursor}
+        )
+        nodes = response["data"]["account"]["users"]["nodes"]
+        users.extend(nodes)
+        page_info = response["data"]["account"]["users"]["pageInfo"]
+        has_next_page = page_info["hasNextPage"]
+        cursor = page_info["endCursor"]
+
+    return users
+
+# define main function for a graphql query of a list of users, prompting the user to select a role from a static list of options, then running a graphql mutation to change all users to that role
+def main():
+    # get list of users
+    users = fetch_users()
+    # print list of users
+    print("List of users:")
+    for user in users:
+        print(f"Role: {user['role']}, Email: {user['email']}, Name: {user['name']}, ")
```

Comment:
Could be better off here just outputting the JSON for `user` here (this ties into my suggestion about the list users query)